src/web/src/components/Posts/Timeline.tsx

```diff
@@ -1,7 +1,7 @@
 import { ReactElement } from 'react';
 import { Container, createStyles } from '@material-ui/core';
 import { makeStyles, Theme } from '@material-ui/core/styles';
-import PostComponent from './Post';
+import PostComponent from './GitHubInfo/Post';
```

Comment:
Why do you want  to move Post into GitHubInfo?

Reply:
I was referring to different PR (#2379, #2367, #2376) and finding what files to move into the `GithubInfo` file. Should have not include that there.

Reply:
ok. I move the file back